core/src/test/java/nu/yona/server/subscriptions/entities/BuddyTest.java

```diff
@@ -1,12 +1,11 @@
 /*******************************************************************************
- * Copyright (c) 2018, 2019 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License, v.
- * 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ * Copyright (c) 2018, 2019 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License,
```

Comment:
2020 now? ;)

Reply:
For new PRs, yes, but this one was created in 2019